src/Resque/Job.php

```diff
@@ -803,4 +803,17 @@ public static function cleanup(array $queues = array('*'))
 
         return $cleaned;
     }
+
+    /**
+     * Update the Delayed Job Status to STATUS_CANCELLED manually
+     *
+     * @param string $JobId
+     */
+
+    public static function cancelJob($JobId) {
```

Comment:
Please use pascalStyle names for variables and methods (`$jobId` instead of `$JobId`).